includes/class-platform-checkout-tracker.php

```diff
@@ -282,12 +282,16 @@ public function checkout_start() {
 	 * Record a Tracks event that the order has been processed.
 	 */
 	public function checkout_order_processed() {
-		$this->maybe_record_event(
-			'order_checkout_complete',
-			[
-				'source' => isset( $_SERVER['HTTP_X_WCPAY_PLATFORM_CHECKOUT_USER'] ) ? 'platform' : 'standard',
-			]
-		);
+		if ( isset( $_SERVER['HTTP_X_WCPAY_PLATFORM_CHECKOUT_USER'] ) ) {
+			$this->platform_checkout_order_processed();
+		} else {
+			$this->maybe_record_event(
+				'order_checkout_complete',
+				[
+					'source' => 'standard',
+				]
+			);
+		}
 	}
```

Comment:
@brianyu28 I'd love it if you could take a look at these changes to the tracking function (and the new `platform_checkout_processed` function) just to check if this looks right 🙏

Reply:
:+1: The tracking function changes look good to me.